packages/block-library/src/more/edit.js

```diff
@@ -74,6 +75,7 @@ export default class MoreEdit extends Component {
 						size={ inputLength }
 						onChange={ this.onChangeInput }
 						onKeyDown={ this.onKeyDown }
+						style={ currentWidth }
```

Comment:
Thanks for the PR. It seems that this doesn't work properly in Latin languages as the input is way too big. I'm not certain what's the correct way to fix this though.

Reply:
Thanks for the review!
I thought it would be better to see it than we couldn't see it. Just because we can't see the text, the function of this block looks incomplete in Japanese.

Reply:
I wonder if there's maybe a way we can design this block differently to avoid this issue. cc @jasmussen @mapk

Reply:
We can absolutely tweak the design to be more inclusive here, I can probably knock something together. 

Do you have a screenshot of what you're seeing, with the block being way too big? Cooking dinner as I write this so did not compile and check the branch.

Reply:
GIF:

![read more](https://user-images.githubusercontent.com/1204802/70516003-83b3bd00-1b36-11ea-81e8-8572ff5a18f2.gif)

Honestly I don't think it's the end of the world. Pending a better design of this, I think this would be a fine interim fix. 👍 👍

Reply:
That looks good yeah, I think while testing the "increase as you type" seemed more severe.